frontend/src/app/account/profile/[id]/page.tsx

```diff
@@ -0,0 +1,157 @@
+"use client";
```

Comment:
What purpose does this page serve?

Reply:
This page has been deleted on my branch, profile page is no longer a dynamic route